### DISCUSSION TITLE
"Talking" xDrip : voice low and "coherent"

### DISCUSSION THREAD

**Discussion by robertrub** [NightscoutFoundation/xDrip#2642] [Q&A]
When phone is connected to the BT of the car, xDrip starts saying BGs (great feature).

The problem is that the voice volume is very low. I have Waze, Spotify and xDrip. The first 2 have a very correct sound level but xDrip can't almost be heard.

How can I raise the xDrip volume ? All I fould was to lower the volumes of the 2 others and push the car audio volume up but then, any notifications are ultra loud.

Another problem is when BGs are stable (not raising or going down). What xDrip says is the number than something that doesn't mean anything there, "Coherent".

Why doesn't xDrip day something easy as "Unchanged" or "stable" ?

**Comment by Navid200** [comment]
There is an open feature request to add Audio focus to speak readings:  https://github.com/NightscoutFoundation/xDrip/issues/2136
When that is done, the xDrip relative volume will be higher when glucose is spoken.

To raise the xDrip speak volume, you can use the media volume of your phone.

When the trend is flat, xDrip spoken readings says the reading followed by "flat".

**Comment by robertrub** [comment]
Hi @Navid200 
Thanks for the answer. Hope that adding _Audio focus_ won't need too much complicated work. It will be a nice add :)

As for "Flat", my xDrip speaks French and does NOT say "Plat" or anything similar to "Flat". Do you think that I can try find "Coherent" in the translations and correct it or is it something à part ?

**Reply by Navid200** [reply]
That is a good feature and I believe it will be added.  I cannot guarantee when.

**Reply by robertrub** [reply]
I found and changed "Cohérente" in Crowdin. I hope it will be validated soon.

**Reply by Navid200** [reply]
Crowdin updates (into xDrip) are very frequent.

**Reply by robertrub** [reply]
Hi. I've made other smaller corrections in the French translations. I don't know who reviews and validates French. The correction from 10 days ago doesn't seem to be confirmed.

**Reply by Navid200** [reply]
Please be patient.  
There is a lot every volunteer has to do.  
It will be done.

**Reply by Navid200** [reply]
@robertrub Can you try the latest Nightly released just a few hours ago?

**Reply by robertrub** [reply]
Thanks a lot. The problem has been solved 👍
